{{ cookiecutter.folder_name }}/.travis.yml

```diff
@@ -17,18 +27,31 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   #Grab all dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coverage pip wheel pytest
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip wheel pytest
   #Launch Conda environment
   - source activate test-environment
   #Install pedl
   - pip install codecov
   - pip install -r requirements.txt
   #Install
-  - python setup.py install
+  - pip install -e .
```

Comment:
I'm a fan of the whole `conda build` stuff for the `hutch-python` tests, but I could see why you might not want that as the template